book/website/collaboration/new-community/new-recruitment.md

```diff
@@ -0,0 +1,524 @@
+        * Provide transparency over interview structure and application assessment criteria where possible 
+    * Include relevant policies clearly in the job advert 
+    * Be transparent about the compensation package
+        * Provide a starting salary or salary range 
+        * Include any statutory or enhanced benefits your organisation may offer such as paternity, maternity and parental leave and/or benefits such as time off for dependents, and annual leave entitlement. 
+    * Directly reference your organisational EDI commitment
+        * Include your organisational Diversity Statement where available.
+        * Include your organisational Values (if applicable). 
+- [ ] Post job advert 
+    * Consider the best places to advertise to reach a diverse pool of candidates
+        * See “ Resources” section for suggestions of advertising venues
+        * If certain demographics are often missing from your pool, utilise spaces that target those groups
+        * Consider reviewing the diversity of the pool attained from previous, similar advertising and seek specific advertising to reach other groups that were underrepresented 
+    * Use a gender decoder to review the advert after you have created it
+    * Leverage your networks, as well as those of others in the organisation, to share the opportunity advertised e.g. Slack, LinkedIn, word of mouth at conferences etc. 
+        * When utilising your networks, consider if there are any groups potentially missing. Consider using alternative spaces (examples listed in Resources section) to reach groups that aren’t in your network. 
+- [ ] Set a suitable closing deadline
+    * Ensure a closing date is included on the advert
+        * Advertise roles minimally for 2-4 weeks where possible
+    * Where possible, stick to the advertised close date. Avoid stating that the role will be closed early if a suitable candidate is identified.  
+    * Consider the day of the week and time of closing your advert. For example, a closing date set on a weekend may be valuable for people who have limited free time.
+    * Be explicit on the time zone which will be used to close the application period 
+    * 
+
+
+### Before job advert closes
+
+
+
+- [ ] Book time(s) for interview panel *[Contributing to this action may be out of scope for hiring managers depending on organisational processes]*
+    * Consider flexibility and other requirements for applicants AND members of the interview panel. For example, ensuring that panel members have sufficient  breaks and that flexibility in scheduling can be offered to applicants - planning should be undertaken in partnership with your HR team.  
+    * Ensure there is a diverse interview panel for every interview - consider gender, race, diversity of thought, interest/impacted group perspective. 
+    * Consider having the option of both in-person or remote interviews for candidates to select from
```

Comment:
It is important to keep in mind here that having some interviews in-person and some online might disadvantage those online, as in-person interviews usually leave a stronger impression and connection to the candidate. It might be better to have an online interview as default, and treat in-person as "reasonable adjustment" option if a candidate has a reason to request it.

I might be wrong here, so feel free to argue against it/disregard.

Reply:
I had the same instinct @MarionBWeinzierl . I like the idea of deciding that one mode (remote/in-person) is the default and offering the other as a resonable adjustment. I'd like to leave it open to the hiring team to decie which is the default, as long as they are covering expenses for in-person!! How does that sound?

Reply:
I agree with @cassgvp here - decide on a clear default, offer reimbursement if people are travelling to attend in person. I personally feel very okay with remote interviews but some teams have different cultures